AsyncDisplayKit/ASCollectionView.h

```diff
@@ -336,6 +336,14 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic) BOOL zeroContentInsets;
 
+/**
+ * Class method to enable a visualization overlay of the number of cells and views in the ASCollectionView.
+ * For app debugging purposes only.
+ *
+ * @param enabled Specify YES to make this debug feature enabled when messaging the ASControlNode class.
+ */
++ (void)setEnableDebugCounts:(BOOL)enable;
```

Comment:
This is fine for now, but soon we should move these kind of features to category definitions in a new file, something like AsyncDisplayKit+Debug.h